external-crates/move/crates/move-vm-runtime/src/execution/vm.rs

```diff
@@ -163,6 +209,7 @@ impl<'extensions> MoveVM<'extensions> {
         function_name: &IdentStr,
         type_arguments: Vec<Type>,
         serialized_args: Vec<impl Borrow<[u8]>>,
+        tracer: Option<&mut MoveTraceBuilder>,
```

Comment:
I like this. I think this assuages the comment higher about whether we wanted `&mut Option<VMTracer>` or `Option<&mut VMTracer>` -- I think I like it the way it is here.